### DISCUSSION TITLE
docs/JsFrame/Vue/%E9%A6%96%E9%A1%B5

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#515] [Announcements]
# docs/JsFrame/Vue/%E9%A6%96%E9%A1%B5

import HomeCard from "@HomeCard"; var cardList = [{ title: "Vue官网", describe: "渐进式的JavaScript框架", imageUrl: "https://v2.cn.vuejs.org/images/logo.svg", openUrl: "https://cn.vuejs...

http://localhost:8080/docs/JsFrame/Vue/%E9%A6%96%E9%A1%B5.html

**Comment by XiaoYang943** [comment]
嘻嘻~
